index.d.ts

```diff
@@ -0,0 +1,108 @@
+declare module 'dicom-parser' {
+ 
+  export type ByteArray = Uint8Array | Buffer;
+
+  export interface Element {
+    tag: string;
+    vr?: string;
+    length: number;
+    dataOffset: number;
+    items?: Element[];
+    dataSet?: DataSet;
+    parser?: ByteArrayParser;
+  }
+
+  export interface DataSet {
+    byteArray: ByteArray;
+    byteArrayParser : ByteArrayParser;
+    elements: {
+      [key: string]: Element;
```

Comment:
Replace `key: string` with `tag: string`, since this is what you used in your other function. Maybe even add a comment saying: `Access Element with tag: xGGGGEEEE`.

Reply:
Good suggestion. Added.